claim.notcoin

```diff
@@ -0,0 +1,8 @@
+[
+    {  
+        "subdomain": "claim.notcoin",  
+        "url":  "claim.notcoin.is-amaz.ing",
```

Comment:
![image](https://github.com/is-amazing/register/assets/166541627/ce9bec84-7179-4c61-a697-d62dce8ae926)

Id:6893146288